### PR TITLE
C++ guide changes

### PR DESCRIPTION
**Pull request by aleksanb**
:dancers:

### CODE REVIEW

File: guides/cpp-intro.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ Understanding of programming languages in general is expected.
 
 # This code example
 
-This guide will use the following code example called blinky to explain how to use C++ with mbed.
-This program flashes a LED on the development board on and off.
+This guide will use the "Hello World! of embedded development, "blinky", to explain how to use C++ with mbed.
```
Comment (stianjensen) [inline guides/cpp-intro.md]: missing " after hello world

File: guides/cpp-intro.md [hunk 2]
```diff
@@ -121,6 +116,35 @@ myled = 1;
 Here the `=` operator is overloaded and is used to set the value of the led.
 So when you change the value of `myled` here you are actually setting the value of the led.
 
+# Printing in mbed c++
+
+In normal c++ one would use `prinft("Hello world\n")` or `cout << "Hello world" << endl` to print strings to the standard output of the program.
```
Comment (stianjensen) [inline guides/cpp-intro.md]: prinft

Comment (stianjensen) [inline guides/cpp-intro.md]: Do we need to show two ways of printing?

File: guides/cpp-intro.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ int main() {
     while(1) { // this is an infinite loop
         myled = 1;
```
Comment (sigvef) [inline guides/cpp-intro.md]: `// Turn led on`?

File: guides/cpp-intro.md [hunk 4]
```diff
@@ -49,27 +49,23 @@ Writing that line will call the constructor of the object.
 In our case we are sending the constructor the parameter `LED1`.
 After this line we have the variable `myled` initialized with the parameter `LED1`.
 
-## Accessing attributes in objects
+## Accessing methods and attributes on objects
```
Comment (sigvef) [inline guides/cpp-intro.md:52]: (y)

File: guides/cpp-intro.md (hunk 2 above)
Comment (cristeahub) [inline guides/cpp-intro.md]: In C++ it's common to use `cout`, however in mbed it is `logger.printf`. if we just want this to be for the mbed-programmer teaching `cout` is not that useful

**Comment by aleksanb** [comment]
Ping

**Pull request merged**